src/modules/threads.ts

```diff
@@ -56,7 +56,16 @@ export default class Threads extends API {
       const fileIndex = await this.schemas.add(schema)
       targetSchema = fileIndex.key
     } else if (schema && typeof schema === 'string') {
-      targetSchema = schema
+      // check if it is one of the default schemas
+      const known = this.schemas.defaultByName(schema)
+      if (known) {
+        // if one of the default, add it or get it to find the hash
+        const added = await this.schemas.add(known)
```

Comment:
is it safe to add even if we've already added? i don't recall?

Reply:
well... they are just content addresses... so yes. we should probably add support on the other side of the api for the default by name